### ISSUE TITLE
Run AdoptOpenjdk tests only

### ISSUE THREAD

**Issue by klepiz** [AdoptOpenJDK/openjdk-tests#1420]
The tests runs the jtreg and the AdoptOpenjdk test, but I want to run AdoptOpenjdk tests only. Is there a way for me to accomplish this? 

thanks :)

**Comment by klepiz** [comment]
I'm interested in `AdoptOpenJDK` tests **only**

**Comment by smlambert** [comment]
Ok I can interpret your request 2 ways, but I think you want the 1st option below:

1) To run test material that are hosted 'in AdoptOpenJDK repos':
- https://github.com/AdoptOpenJDK/openjdk-systemtest  (use TARGET=system)
- https://github.com/AdoptOpenJDK/openjdk-tests/tree/master/functional (use TARGET=special.functional)

2) To run the test material that 'AdoptOpenJDK runs' to verify their binaries:
- openjdk tests from openjdk project (use TARGET=openjdk)
- system tests from openjdk-systemtest and openj9-systemtest repo (use TARGET=system)
- functional tests from openjdk-tests and openj9 repo (use TARGET=functional)
- external tests pulled from various large Java application repos and microprofile tcks (scala,tomcat,jenkins, etc) (use TARGET=external)
- perf tests pulled from various benchmark repos (use TARGET=perf)

In other words, AdoptOpenJDK uses test material from a variety of locations to verify its binaries.  More details here: https://medium.com/adoptopenjdk/the-first-drop-introducing-adoptopenjdk-quality-assurance-aqa-v1-0-fe09f10ced80

So, the steps to run the system tests:
export TEST_JDK_HOME=/locationOfJDKUnderTest
git clone https://github.com/AdoptOpenJDK/openjdk-tests.git (to /testLocation)
cd openjdk-tests
./get.sh -t /testLocation/openjdk-tests
cd TestConfig
export BUILD_LIST=system
make -f run_configure.mk
make compile
make _sanity.system

**Comment by klepiz** [comment]
I set $BUILD_LIST and then make -f run_configure. But I have this issue when I run `make compile` :/ 
`get-source:
     [exec] Didn't find password for CVSROOT ':pserver:anoncvs@sourceware.org:/cvs/mauve'.`

**Comment by smlambert** [comment]
Perhaps we can connect tomorrow via AdoptOpenJDK [Slack #testing channel](https://adoptopenjdk.slack.com/messages/C5219G28G), and step through it.  

alternatively please share the full output of your compile step, and please share printenv output, so I can see if there are env vars that are set but wrong.

I will verify tonight that the instructions I shared work locally on my machine.

**Comment by smlambert** [comment]
You can also see the console output of a Grinder job that runs sanity.system target to see the commands that are run to achieve it:

https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/961/consoleFull

**Comment by smlambert** [comment]
I have verified the instructions shared to run system tests locally.  I also captured the steps in a [1 minute recording](https://youtu.be/EIjWERDHBdY).  If you follow those steps without variance and still are unable to run the system tests, please re-open this issue (and share output of printenv to see what may be running interference).

**Comment by klepiz** [comment]
yeah, it already works :) thanks for your help
